docs/iot/deployment.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Deploy .NET apps to Raspberry Pi
+description: Learn how .NET can be used to build applications for IoT devices and scenarios.
+author: camsoper
+ms.author: casoper
+ms.date: 11/2/2020
+ms.topic: overview
+ms.prod: dotnet
+---
+
+# Deploy .NET apps to Raspberry Pi
+
+Deployment of .NET apps to Raspberry Pi is the identical to that of any other platform. Your app can run as *self-contained* or *framework-dependent* deployment modes. There are advantages to each strategy. For more information, see [.NET application publishing overview](/core/deploying).
+
+## Deploying a framework-dependent app
+
+To deploy your app as a framework-dependent app, complete the following steps:
+
+1. [!INCLUDE [ensure-ssh](includes/ensure-ssh.md)]
+
+1. Install .NET on the Raspberry Pi using the [dotnet-install scripts](/dotnet/core/tools/dotnet-install-script). Complete the following steps from a Bash prompt on the Raspberry Pi (local or SSH):
+    1. Run the following command to install .NET:
+
+        ```bash
+        curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin
+        ```
+
+        > [!NOTE]
+        > This installs the latest version. If you need a specific version, add `--version <VERSION>` to the end, where `<VERSION>` is the specific build version.
+
+    1. To simplify path resolution, add a `DOTNET_ROOT` environment variable and add the *.dotnet* directory to `$PATH` with the following commands:
+
+        ```bash
+        echo 'export DOTNET_ROOT=$HOME/.dotnet' >> ~/.bashrc
+        echo 'export PATH=$PATH:$HOME/.dotnet' >> ~/.bashrc
+        source ~/.bashrc
+        ```
+
+    1. Verify the .NET installation with the following command:
+
+        ```bash
```

Comment:
The `dotnetcli` devlang will give you better colorization of the commands.

```suggestion
        ```dotnetcli
```